global.json

```diff
@@ -1,6 +1,5 @@
 {
   "sdk": {
-    "version": "9.0.100-preview.7.24407.12",
```

Comment:
What's the significance of removing `rollForward`?

Reply:
RC2 preview was causing trouble on my machine.